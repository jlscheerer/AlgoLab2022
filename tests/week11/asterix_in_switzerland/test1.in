30
5 11
-11500 -263274 -197300 215472 -171226
2 1 159585
4 3 117353
3 2 25308
1 4 140817
2 3 150971
4 2 156503
3 4 65212
4 0 53306
3 0 123112
0 1 33829
3 1 39777
10 12
-288 133 -106131 -1000000 -149 -93866 56 -1000000 -1000000 -1000000
6 7 110345
2 7 217382
5 8 77801
9 7 161179
7 8 203737
1 0 71498
3 7 241409
7 3 189560
7 4 242044
5 6 267028
5 4 72641
8 4 250185
6 13
-1000000 -85471 -1000000 -1000000 115732 -71629
3 0 76437
2 0 107656
5 1 46207
4 1 9513
4 0 116699
0 5 101252
2 3 34766
3 2 38250
1 5 38996
1 0 90833
3 1 77245
1 4 135225
2 5 191744
6 9
-1000000 -3060 -3103 -126656 1061 -78549
2 4 138059
3 1 80555
5 1 22755
0 5 26937
0 1 44091
3 2 134862
4 0 26429
5 4 111362
0 4 125653
5 13
-1000000 23599 -226161 17816 -59569
3 4 125212
1 0 57105
1 4 26101
2 4 4858
2 1 75077
0 3 35258
0 4 49847
1 2 32350
4 3 106212
2 3 91800
0 2 130058
2 0 20430
3 1 93211
8 10
-1251 -1000000 -1000000 -1000000 -28710 -1000000 30710 -1000000
6 4 289797
5 7 55447
2 4 55372
7 4 30599
7 5 79555
1 2 193020
3 2 252499
7 2 178108
2 0 76169
3 5 274157
8 11
-82702 -1000000 -8183 -26207 -586 -431151 -11995 -1000000
1 2 274865
6 5 43112
3 2 215181
2 5 3695
7 6 259132
6 1 113243
5 3 148045
5 2 52934
3 6 231238
1 6 286927
5 1 238367
8 8
-1000000 -22612 -902 -379 -1000000 12475 -8348 -1000000
5 6 112401
4 2 15542
4 7 249868
4 5 275965
0 5 31820
7 3 78075
7 1 184731
0 6 73377
10 11
-1000000 -604591 -1000000 -88 -225088 -58 -16069 330050 -81 -690364
0 7 100526
6 8 176978
2 5 33855
6 4 158117
9 7 330974
1 2 217504
7 4 330056
4 7 99853
2 7 63345
9 0 36214
6 1 314578
7 11
155039 -13691 -23773 -1000000 -607490 -1000000 -216191
4 0 123977
6 4 115875
5 2 152218
1 3 193740
0 5 130659
5 6 126487
2 3 8037
1 6 189878
3 4 220526
1 0 54684
0 2 24388
10 13
175941 -135 -48 -243960 -733 -27 -25157 -281143 -90931 459568
3 9 130281
9 0 265122
7 4 162823
8 3 27360
9 2 79839
8 5 233503
6 1 107304
8 7 83703
7 2 38183
9 8 114615
6 4 198317
7 6 173617
0 8 175947
7 10
-206445 -11 -523578 -1000000 383126 -978280 -417621
3 4 229055
0 2 95720
0 3 145704
4 0 141485
4 5 2702
6 0 80259
2 6 179082
4 3 238945
3 0 170944
5 2 229900
6 12
114 -23942 -34102 194264 -211482 -1000000
5 2 132433
1 0 3035
3 5 116817
0 5 20479
5 0 133970
0 2 64899
2 0 15434
4 2 41747
4 3 73497
3 0 153414
1 5 23866
1 3 64053
7 14
-48165 -53642 31357 -959527 -1000000 -1000000 -126248
3 6 198989
5 2 30269
3 2 192172
3 1 57527
1 4 62174
5 4 136575
0 2 1230
2 0 83710
1 5 124084
6 5 16851
4 5 171689
4 0 52889
3 0 33166
6 0 209894
10 15
-1000000 192646 -1000000 -7224 -217 -1000000 -1000000 -16056 -164516 -61215
0 7 120440
1 9 121834
0 5 274088
1 4 25498
9 8 261598
7 1 224732
5 1 228887
6 1 188676
2 0 276384
5 7 41677
8 4 156169
7 0 64473
8 1 113364
5 4 120871
1 8 149921
19 24
-1000000 -1000000 -65 -114 -72 -1000000 -1000000 245516 -373878 -45 -1000000 -1000000 -563480 -1000000 -305 -1000000 -31937 -1000000 -912140
18 1 125560
13 7 347932
10 9 118171
12 3 230878
10 1 228371
0 15 74426
18 12 288059
5 11 44539
8 4 94067
18 0 288962
16 10 17966
1 3 103977
17 6 278510
11 1 329698
6 14 314990
11 10 134692
8 3 142136
0 11 313586
11 16 61742
5 12 160181
15 5 253147
7 8 266784
8 0 102363
10 15 334944
14 24
151814 -1000000 -1000000 -349610 -677632 -1000000 -1000000 -619501 -1000000 -76898 -107 -793342 16154 -560080
2 9 158741
13 5 45850
6 9 14237
3 13 3547
11 2 221576
0 9 242531
5 0 237236
3 4 36057
1 0 86946
12 9 123475
3 1 47818
9 7 31532
6 11 169803
4 9 198383
11 13 233095
8 13 155056
7 0 38938
6 10 194240
4 6 6296
7 13 195426
1 6 63039
13 9 200556
4 1 208259
4 5 172216
14 25
-103420 -1000000 -161612 -32707 -309400 -665 -1000000 -57724 -13168 -602843 -10437 -1000000 -125756 -1000000
4 3 243820
9 6 64615
11 10 7166
2 12 164001
11 2 150957
7 12 51590
6 11 44979
2 9 13066
2 10 144421
6 4 169249
4 1 204708
12 8 20014
13 2 27769
1 4 64728
2 0 198666
11 13 143273
8 4 229424
13 1 66007
3 1 23568
13 7 13209
4 11 104901
10 13 42156
4 5 225472
1 10 137622
0 1 225620
19 33
-3307 -1000000 -1000000 72405 -488034 -764375 -636401 -560181 -138 -40861 -1000000 -87683 -719704 -123594 -345784 -1000000 -8490 -705 -327086
7 11 164371
3 0 182635
14 5 231013
13 3 5753
5 15 42700
3 16 78626
2 0 91437
15 9 211061
12 13 202099
18 17 145186
1 0 104643
4 3 104506
11 15 6059
2 17 142859
2 12 225255
9 12 228347
15 5 77849
4 5 186931
11 6 230367
13 16 12248
12 14 4730
18 14 180129
12 1 252405
2 8 178511
6 4 138088
5 0 38564
3 13 260318
18 6 40059
9 10 80189
13 11 1056
10 1 130184
6 11 226425
7 3 231288
20 35
-364050 -3635 -1000000 -1000000 -1000000 483489 -416 -404621 -1000000 -1000000 201 -1000000 -347039 -727823 -1000000 9641 -1000000 -177778 -243466 -253628
0 16 31756
18 19 78213
18 12 67223
0 14 197569
13 11 55750
3 5 128770
14 1 180538
9 6 200813
5 8 227808
7 8 126413
16 15 67582
8 17 38677
13 1 71816
2 19 245035
3 8 33251
2 10 41126
19 10 163202
2 7 102477
2 18 63232
7 13 62627
2 4 46367
12 16 79822
11 4 249742
1 15 36808
2 1 2375
18 3 44594
13 17 247089
14 11 54465
4 17 160897
12 13 243501
17 13 22863
10 16 230303
15 7 242317
16 4 52909
5 10 255689
10 37
-4176 -113086 -451735 63420 -5499 -412956 10717 -28125 -152388 7530
5 2 56982
1 2 68462
9 5 77223
8 7 6715
1 5 93339
3 2 43040
6 7 9912
4 0 26099
1 4 58379
7 0 71295
9 0 60921
1 8 90987
1 6 19430
8 1 97076
8 6 26744
6 5 93851
8 2 1349
2 3 84759
5 0 28798
2 4 41799
4 2 96049
0 6 16943
8 3 87664
8 4 79076
5 7 34758
5 8 69881
0 5 6998
0 2 49726
3 7 31685
7 2 17580
0 4 13529
6 3 86265
1 0 34842
9 7 59313
6 0 47689
2 8 4152
7 4 68371
14 42
450349 -19228 -1000000 -433788 -1000000 -730458 -65 -1000000 -1000000 -1000000 -8378 -422036 -1000000 27905
1 11 34447
13 5 17825
8 3 65227
3 4 114829
13 7 109980
1 8 22409
11 3 144651
13 6 123860
5 6 64729
0 13 110738
7 3 133357
3 12 26033
4 10 93876
1 9 91282
10 4 132211
12 11 46246
3 7 114874
2 0 86498
0 10 36819
11 8 46459
2 10 129210
1 2 84852
0 7 131631
9 12 97201
12 2 12441
0 1 1875
3 13 65866
3 6 91022
9 7 145141
8 5 98780
9 3 92914
4 7 148558
5 3 151451
8 13 143838
11 10 87938
9 6 70444
11 5 21595
0 8 145984
9 1 45329
0 3 24182
2 12 106179
0 2 118525
10 20
-877952 246782 -602099 -9 -19586 -1000000 -305408 333562 -228630 -46523
8 1 19657
0 3 132516
1 9 97624
9 4 130862
2 6 43979
7 5 136584
7 9 131360
0 8 79764
7 1 65626
4 5 143661
8 7 86724
6 7 32352
2 7 144508
5 8 57427
4 0 76296
1 7 159638
9 6 39262
1 6 149059
8 4 98902
9 8 117229
13 32
11623 -1000000 -267651 258700 -57867 -9161 -437945 -5898 -1000000 -79832 -668879 -665788 -6078
4 2 91966
9 0 3630
2 3 101404
11 3 118191
6 1 37447
10 2 153641
9 3 15748
4 7 23521
4 3 61982
2 1 22096
12 6 32732
1 7 101677
7 8 8651
3 5 19299
6 9 148041
10 1 65358
6 0 62782
0 5 124315
3 6 74536
8 5 148696
3 8 131669
7 6 117333
8 7 43670
5 12 29023
6 2 82663
6 5 40639
8 9 66220
12 8 95227
9 5 70665
3 1 33206
0 3 24685
11 2 64566
17 36
-1000000 -6759 -771607 -26450 -356937 -245 -33 -211599 160612 -452667 -1000000 -40886 1230 -16992 -227260 -1000000 -1000000
3 7 15953
8 0 76725
11 9 19633
16 7 76157
0 12 180770
15 0 56229
10 12 158888
0 6 120013
16 15 173508
11 12 166859
15 5 219317
7 14 24802
16 10 12591
14 1 75016
2 13 32268
15 4 78363
3 12 124140
8 6 56398
3 15 136973
12 10 1605
1 0 4862
14 15 17771
0 11 190750
4 3 38997
14 12 87519
7 12 136628
13 4 27
2 15 149998
13 14 55827
9 14 150829
8 2 27497
2 12 142559
12 1 142165
14 4 110630
9 10 68455
2 1 43309
13 24
-1000000 159447 -39768 -2298 -105259 -33809 -62430 -477675 -1000000 -1000000 -158632 -493275 -119457
9 11 38365
7 9 24785
7 11 60344
8 3 29779
11 3 209605
8 4 97098
6 2 76741
10 2 93517
12 0 62384
11 7 75689
5 8 144335
7 0 74445
1 11 162420
2 6 119560
7 6 106057
11 12 34574
0 6 218368
4 5 108396
7 10 213589
10 9 29980
3 4 56596
4 7 72299
0 11 153173
9 0 97083
17 30
-259466 -1000000 -2068 -1000000 -16627 -1000000 -1000000 -2136 -1000000 -1000000 -107568 -1000000 -546147 -1000000 -1000000 -30245 268259
14 1 236229
6 1 60890
3 0 180510
3 4 153791
5 10 143422
3 10 160685
16 10 231275
0 9 61877
13 15 20914
11 15 155026
5 8 139465
16 15 17653
1 16 22628
1 11 180712
1 3 118059
1 7 250290
14 13 125329
5 14 185394
0 7 167327
2 5 77672
14 8 148489
3 5 103526
2 13 17364
16 12 193051
12 5 144735
0 2 24698
11 14 224026
1 12 45347
14 12 82186
8 14 129190
15 31
-67083 -1000000 -1000000 -1000000 -1000000 -1000000 -52562 -56460 -986528 -983875 -582 -6016 -43349 -1000000 78096
6 3 6266
6 0 79833
13 1 76022
13 4 203498
12 6 16974
1 7 109060
4 10 171053
14 12 185318
5 1 116286
9 10 136203
4 3 177058
8 13 173785
3 5 162684
11 6 212484
1 14 148587
9 6 128913
8 7 139
0 12 2220
7 12 164760
7 14 152470
8 11 132108
7 0 159550
6 4 101928
6 9 178199
6 8 213464
4 8 136721
3 6 199175
2 1 22610
7 13 154707
11 8 184243
2 5 28222
17 25
-1000000 -1000000 -1000000 -1000000 -1000000 -112291 399 -1000000 -10871 -356 -1000000 -1000000 -197880 -1097 -407260 -20412 -1000000
2 3 10840
0 3 109958
15 2 243917
4 9 197466
7 10 173598
14 2 52477
16 1 104858
0 7 182703
7 13 201371
14 6 71133
14 0 170212
2 8 162001
7 3 158576
1 6 265742
6 13 93246
11 4 252261
3 13 130377
9 2 157879
4 3 93686
4 11 159027
10 14 250760
11 15 86752
16 12 118006
10 6 132755
16 8 189050
20 43
-1000000 -1000000 -1000000 -1000000 -1000000 -190205 157209 -71034 -522 -1278 -1000000 -705772 -261464 -32 -3620 -1000000 -4506 -1405 -10017 -1000000
15 9 61390
18 17 76830
10 14 89035
2 8 73696
18 16 205386
15 0 87665
5 18 168272
17 0 53462
12 18 77368
15 13 205146
0 3 28513
14 5 126475
18 12 128301
12 6 212280
19 1 109718
5 0 36381
6 2 215462
2 12 83145
0 11 166097
1 18 160301
12 8 202594
10 8 204999
6 5 125226
17 8 48588
4 10 130048
17 13 122249
9 14 58932
0 8 68778
3 9 111944
1 5 105462
1 3 204611
7 6 149877
14 1 1489
6 8 55510
12 13 122559
11 6 94329
11 2 95889
4 12 22427
12 4 108486
11 9 61545
7 4 23222
17 9 60523
3 18 170182
