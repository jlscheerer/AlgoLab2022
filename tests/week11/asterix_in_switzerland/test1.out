yes
no
no
no
no
yes
no
yes
yes
no
yes
no
yes
no
no
no
yes
no
no
yes
no
no
yes
no
yes
no
no
yes
no
no
